search/includes/classes/commands/class-corecommand.php

```diff
@@ -45,6 +46,11 @@ private function _shift_version_after_index( $assoc_args ) {
 				WP_CLI::error( sprintf( 'Error activating next version: %s', $result->get_error_message() ) );
 			}
 
+			if ( ! $skip_confirm ) {
+				WP_CLI::confirm( '⚠️  You are about to remove a previously used index version. It is advised to verify that the new version is being used before continuing. Continue?' );
+			}
+
+
```

Comment:
I'd remove one line break so we don't have two

Reply:
done:)